pynamodb_attributes/timestamp.pyi

```diff
@@ -5,15 +5,12 @@ from ._typing import Attribute
 
 
 class TimestampAttribute(Attribute[datetime]):
-    def __get__(self, instance: Any, owner: Any) -> datetime: ...
```

Comment:
This should've been at the very least two overloads
```
def __get__(self, instance: TimestampAttribute, owner: Any) -> TimestampAttribute: ...
def __get__(self, instance: Any, owner: Any) -> datetime: ...
```
but actually we shouldn't, since the generic `Attribute[T]` already does that.

Reply:
(That's how the rest of the attributes' typestubs are defined.)